.github/dependabot.yml

```diff
@@ -26,49 +26,13 @@ updates:
     open-pull-requests-limit: 20
 
   - package-ecosystem: nuget
-    directory: /examples/AspNetCoreMvc
+    directory: /examples/cli
```

Comment:
CLI makes me think about "command-line interface", not "client".

Reply:
it is a "CLI app" 😉 Do you want to rename it to `client` or `httpget`?

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1708/commits/cc20430edf4e6863964f62ca94727b6891599792